inst/ssh/github_setup.sh

```diff
@@ -1,10 +1,10 @@
-chmod 600 /root/.ssh/id_rsa
+chmod 600 /root/.ssh/id_ed25519
```

Comment:
What was the reason for this change?

Reply:
I think this isn't actually relevant.  I thought it'd be good to name the identity file corerctly depending on the encryption (RSA vs. ED25519), but I don't think it's necessary.

Reply:
I can remove this addition if you want, unless you think there is any benefit to having a different key option?